catalog/spatial/registry/registry-report-action-provider/src/test/java/org/codice/ddf/registry/report/action/provider/RegistryReportActionProviderTest.java

```diff
@@ -424,10 +424,10 @@ private void configureSecureActionProvider() {
   private void configureActionProvider(
       String protocol, String host, String port, String contextRoot) {
 
-    setProperty("org.codice.ddf.system.hostname", host);
-    setProperty("org.codice.ddf.system.httpPort", port);
-    setProperty("org.codice.ddf.system.httpsPort", port);
-    setProperty("org.codice.ddf.system.protocol", protocol);
+    setProperty("org.codice.ddf.external.hostname", host);
```

Comment:
❓ Should we use the constants for each of these?

Reply:
Fixed